package.json

```diff
@@ -35,6 +35,7 @@
     "@whi/stdlog": "^0.3.3",
     "concat-stream": "^2.0.0",
     "json-stable-stringify": "^1.0.1",
+    "portscanner": "^2.2.0",
```

Comment:
Can this be a dev-dependency instead? Looked like it was only used in tests

Reply:
It should be dev-dependency